src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/abs/EngineGeneralTest.java

```diff
@@ -1811,17 +1811,19 @@ public void joinThreeQueriesTest() throws DatabaseEngineException {
     @Category(SkipTestCockroachDB.class)
     // unimplemented in CockroachDB: views do not currently support * expressions
     // https://github.com/cockroachdb/cockroach/issues/10028
-    public void createViewTest() throws DatabaseEngineException {
+    public void createAndDropViewTest() throws DatabaseEngineException {
         test5Columns();
 
         try {
-            engine.executeUpdate("DROP VIEW " + quotize("VN", engine.escapeCharacter()));
+            engine.dropView("VN");
         } catch (final Throwable a) {
         }
```

Comment:
I know this was already like this, but it is weird we are dropping the view before creating. If the goal is to assert that it throws an exception, then we should assert on it, otherwise this is not doing anything.